menyer/beancrush/beancrush_style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Kufam&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Nova+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bungee+Shade&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Codystar&display=swap');
+
+:root {
+    --custom-darkgray: rgba(112, 112, 112, 0.534);
+    --custom-lightgray: rgba(165, 165, 165, 0.534);
+    --custom-black: rgb(68, 69, 69);
+    --custom-white: rgb(231, 227, 219);
+    --custom-pink-light: rgb(236, 136, 206); 
+    --custom-pink-dark: rgb(195, 50, 152);
+    --custom-blue-light: rgb(185, 227, 240);
+    --custom-blue-dark: rgb(96, 199, 225);
+
+}
+
+#header_image {
+    width: 64px;
+    height: 64px;
+    margin: auto;
+}
+
+body {
+    background-image: url("img_background_concrete.jpg");
+}
+
+section {
+    background-image: url("img_background_scales.jpg");
+    border: 2px solid var(--custom-pink-light);
+    border-radius: 6px;
+    box-shadow: 0 0 12px var(--custom-pink-dark);
+    margin: 25px 150px;
+}
+
+h1 {
+    font-family: "Codystar";
+    font-size: 70px;
+    text-align: center;
+    color: var(--custom-blue-light);
+    text-shadow: 0 0 6px var(--custom-blue-dark);
+    border: 2px solid var(--custom-pink-light);
+    box-shadow: 0 0 12px var(--custom-pink-dark);
+    border-radius: 12px;
+    width: 700px;
+    margin: auto;
+    margin-top: 50px;
+    margin-bottom: 25px;
+}
+
+h2, h3 {
+    font-family: "Nova Round";
+    color: var(--custom-white);
+    text-align: center;
+    margin-bottom: 4px;
+    margin-top: 6px;
+}
+
+h4, p {
+    color: var(--custom-white);
+    font-family: "Kufam";
+    margin: 4px;
+    text-shadow: 1px 1px 1px var(--custom-black);
+}
+
+p {
+    width: 1000px;
+}
+
+.food_item_1, .food_item_2 {
+    height: 42px;
+    margin: 10px;
+    padding-top: 2px;
+    padding-left: 12px;
+    padding-right: 12px;
+    border: 2px dotted var(--custom-blue-light);
+    border-radius: 12px;
+    padding: 25px;
+    box-shadow: 0 0 8px var(--custom-blue-dark);
+}
+
+.food_item_1 {
+    background-color: var(--custom-darkgray);
+}
+
+.food_item_2 {
+    background-color: var(--custom-lightgray);
+}
```

Comment:
Could be replaced by using the psedo-class:

```
.food-item:nth-child(even) {var(--custom-darkgray);}
.food-item:nth-child(odd) {var(--custom-lightgray);}
```

This way you don't need to manually place `food_item_1` and `food_item_2` everywhere. Which is ok for now but if you were to pull the menu data from a database it would be more cumbersome.